dim_checks/moz-fx-data-shared-prod/telemetry/cohort_daily_statistics/dim_checks.yaml

```diff
@@ -0,0 +1,88 @@
+dim_config:
+  owner:
+    email: lvargas@mozilla.com
+    slack: lvargas
+  alerts_enabled:
+    enabled: true
+    notify:
+      channels:
+        - dim_test_run
+  tier: tier_1
+  dim_tests:
+    - type: table_row_count
+      params:
+        condition: ">= 1000"
+    - type: not_null
+      params:
+        columns:
+          - cohort_date
+          - activity_date
+          - activity_segment
+          - app_version
+          - device_model
+          - locale
+          - normalized_app_name
+          - normalized_channel
+          - normalized_os
+          - normalized_os_version
+          - os_version_major
+          - os_version_minor
+    - type: column_length
+      params:
+        columns:
+          - country
+        condition: "= 2"
+    - type: value_in_set
+      params:
+        columns:
+          - normalized_channel
+        expected_values:
+          - nightly
+          - aurora
+          - release
+          - Other
+          - beta
+          - esr
+    - type: value_in_set
+      params:
+        columns:
+          - activity_segment
+        expected_values:
+          - infrequent_user
+          - other
+          - casual_user
+          - regular_user
+          - core_user
+    - type: value_in_set
+      params:
+        columns:
+          - normalized_app_name
+        expected_values:
+          - "Firefox iOS"
+          - Fenix
+          - "Focus Android"
+          - "Focus iOS"
+          - "Firefox Desktop"
+          - "Fenix BrowserStack"
+          - "Firefox iOS BrowserStack"
+    - type: value_in_set
+      params:
+        columns:
+          - normalized_os
+        expected_values:
```

Comment:
Should NULL values generate an alert?

Reply:
Is there a valid reason why a null values would be allowed/expected at this stage of the pipeline? I assume since this is after normalization this value should be set. Otherwise, normalization did not work correctly. Unless, I'm making wrong assumptions about this here. @gkabbz what's your view on this?

Reply:
I'm thinking of the cases when the OS is NULL or unavailable in the telemetry ping. Since nulls are allowed in the table schema, if we track them we would be getting continuous alerts.
Unless the plan is that `null` values generate alerts so we can feedback the owners who would solve it until the alerts dissapears?

Reply:
In this case, I believe this covers it quite well:
> Unless the plan is that null values generate alerts so we can feedback the owners who would solve it until the alerts dissapears?